code/game/mecha/mecha.dm

```diff
@@ -102,6 +102,8 @@
 
 
 
+	var/occupant_sight_flags = 0 //sight flags to give to the occupant (e.g. mech mining scanner gives meson-like vision)

```

Comment:
I added the occupant_sight_flags to mecha if people want to add mech or mech equipment that give with special vision. Currently only used by the mining scanner.